devel/retdec/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        avast retdec 4.0 v
+revision            0
+conflicts           ${name}-devel
+
+categories          devel
+license             MIT
+platforms           darwin
+maintainers         nomaintainer
+
+description         RetDec is a retargetable machine-code decompiler based on LLVM.
+long_description    ${description}. The decompiler is not limited to any particular \
+                    target architecture, operating system, or executable file format.
+
+checksums           rmd160  305e901b37441337d5c0dada1ccc5e737831ef1e \
+                    sha256  b0e3172d9d232899b71dac41446592c03abe089b8e9c7c1a9c8e8bfe92cf8b49 \
+                    size    8732003
+
+patch.pre_args      -p1
+patchfiles          patch-python3.diff \
+                    patch-yara-syntax-error.diff
+
+post-patch {
+    reinplace "s|\${CMAKE_INSTALL_PREFIX}|${destroot}/\${CMAKE_INSTALL_PREFIX}|" ${worksrcpath}/CMakeLists.txt
+    reinplace "s|\${CMAKE_INSTALL_PREFIX}|${destroot}/\${CMAKE_INSTALL_PREFIX}|" ${worksrcpath}/support/CMakeLists.txt
+}
+
+compiler.cxx_standard   2017
+use_xcode           yes
```

Comment:
why? I have successfully build it on VM without xcode :)

Reply:
I didn't add this, it was already in your commit - but it's certainly a good question why this would be needed 😉

Reply:
Indeed!